src/main/java/org/assertj/core/presentation/StandardRepresentation.java

```diff
@@ -138,6 +139,7 @@ public String toStringOf(Object object) {
     if (object instanceof Map<?, ?>) return toStringOf((Map<?, ?>) object);
     if (object instanceof Tuple) return toStringOf((Tuple) object);
     if (object instanceof MapEntry) return toStringOf((MapEntry<?, ?>) object);
+    if (object instanceof Method) { return ((Method) object).toGenericString(); }
```

Comment:
I am not sure if this makes sense to be part of this PR or not. Does it even make sense to be included in AssertJ.

This one helped me a lot to figure out what exactly was the problem in the failing tests.